internal/cmd/local/docker/secret_test.go

```diff
@@ -2,17 +2,15 @@ package docker
 
 import (
 	"testing"
-
-	"github.com/google/go-cmp/cmp"
 )
 
 func Test_Secret(t *testing.T) {
-	exp := `{"auths":{"my-registry.example:5000":{"username":"tiger","password":"pass1234","email":"tiger@acme.example","auth":"dGlnZXI6cGFzczEyMzQ="}}}`
+	exp := `{"auths":{"my-registry.example:5000":{"auth":"dGlnZXI6cGFzczEyMzQ=","email":"tiger@acme.example","password":"pass1234","username":"tiger"}}}`
 	act, err := Secret("my-registry.example:5000", "tiger", "pass1234", "tiger@acme.example")
 	if err != nil {
 		t.Fatal(err)
 	}
-	if d := cmp.Diff(exp, string(act)); d != "" {
```

Comment:
Not a fan of the `Diff` function?

Reply:
Mainly because when I made it fail, I couldn't tell what was wrong. Seeing them one above the other was much easier.